src/libs/routing/utils.ts

```diff
@@ -113,15 +23,11 @@ export const validArrayOf = (schema: v.GenericSchema<string, any>) => {
   return v.array(schema);
 };
 
```

Comment:
Passing the type improves inferring types with arrays, instead of just showing `any[]`

```
export const validArrayOf = <T>(schema: v.GenericSchema<string, T>) => {
  return v.array(schema);
};
```